public/app/features/dimensions/editors/ResourceCards.tsx

```diff
@@ -21,7 +21,7 @@ interface CellProps {
   };
 }
 
-function Cell(props: CellProps) {
+const MemoizedCell = memo(function Cell(props: CellProps) {
```

Comment:
was it clear whether or not this issue was caused by React18 in your troubleshooting? just curious

Reply:
Not really clear tbh, I was curious too if something is different with `memo()` in React18 but couldn't find anything specific. I've just always used it at component level..